### DISCUSSION TITLE
Forward messages from Telegram Bots

### DISCUSSION THREAD

**Discussion by weskerty** [42wim/matterbridge#2097] [Help / Q&A] [answered]
Currently the bot ignores messages from all bots. Is there a way to change it?

**Comment by Mikaela** [comment, marked as answer]
The bot is not ignoring other bots, it's not getting messages from other bots at all since Telegram has decided to not let bots see each other. Thus there is nothing Matterbridge can do about it.

- https://core.telegram.org/bots/faq#why-doesn-39t-my-bot-see-messages-from-other-bots

**Reply by weskerty** [reply]
Thanks for the reply. A solution would be to use a normal user account for Matterbridge? Currently can that be done?
